api/pkg/apis/v1alpha1/managers/stage/stage-manager.go

```diff
@@ -657,7 +657,7 @@ func (s *StageManager) HandleTriggerEvent(ctx context.Context, campaign model.Ca
 		}
 
 		for k, v := range outputs {
-			if !strings.HasPrefix(k, "__") {
+			if !(strings.HasPrefix(k, "__") || strings.HasPrefix(k, "header.")) {
```

Comment:
@msftcoderdjw Do we want to remove all header in the output or in the stage history?

Reply:
I prefer to removing all headers in stage history for now and keep them in the output to avoid the impact of existing providers. We can wait for more customer signals if we want to log more headers. Like logical app, they provide options to include Authorization header - https://learn.microsoft.com/en-us/azure/logic-apps/logic-apps-securing-a-logic-app?tabs=azure-portal#include-authorization-header-in-request-or-http-webhook-trigger-outputs.